packages/react-components/src/radio/src/RadioGroup.tsx

```diff
@@ -158,12 +165,12 @@ export function InnerRadioGroup(props) {
     const { groupProps, itemProps } = useGroupInput({
         cssModule: "o-ui-radio-group",
         role: "radiogroup",
-        keyProp: RadioKeyProp,
-        value,
-        defaultValue,
+        // keyProp: RadioKeyProp, //TODO: why is this passed but not used?
```

Comment:
why are we forwarding useless props? GroupInput doesnt use them

Reply:
Bad refactor I guess, remove them. I think useGroupInput used to "spread" the rest.

Reply:
done